File: 3668-RestoreFinishingOrder/javascript/solution.js
// Last updated: 3/6/2026, 12:38:57 AM
/**
 * @param {number[]} order
 * @param {number[]} friends
 * @return {number[]}
 */
var recoverOrder = function(order, friends) {
    let ans = []
    for(let i = 0; i < order.length; i++){
        if(friends.includes(order[i])){
            ans.push(order[i])
        }
    }
    return ans
};

File: 3668-RestoreFinishingOrder/cpp/solution.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

class Solution {
public:
    vector<int> recoverOrder(vector<int>& order, vector<int>& friends) {
        vector<int> ans;
        for (int i : order) {
            if (find(friends.begin(), friends.end(), i) != friends.end()) {
                ans.push_back(i);
            }
        }
        return ans;
    }
};


int main(){
    
    return 0;
}